.github/workflows/build-and-test.yaml

```diff
@@ -41,6 +41,18 @@ jobs:
           name: manager
           path: bin/manager
 
+  release-tarballs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: "1.16.5"
+
+      - uses: actions/checkout@v2
+
+      - run: make release-tarballs
+      - run: ls -alh ./release
```

Comment:
debugging?

Reply:
"did I actually get 5 tarballs out with nonzero size", yeah

Reply:
speaking of, I'm kinda weirded out by the size differences - locally my builds are in the 17-29M range, and these are 12-14M

Different default compression algorithms?